actions/utils.ts

```diff
@@ -328,3 +329,35 @@ export function sendSlack(message: string): boolean {
   });
   return true;
 }
+
+/**
+ * Attempts to verify that the contract at the given address implements the interface of the `ComposableCoW`
+ * contract. This is done by checking that the contract contains the selectors of the functions that are
+ * required to be implemented by the interface.
+ * 
+ * @remarks This is not a foolproof way of verifying that the contract implements the interface, but it is
+ * a good enough heuristic to filter out most of the contracts that do not implement the interface.
+ * 
+ * @dev The selectors are:
+ * - `cabinet(address,bytes32)`: `1c7662c8`
+ * - `getTradeableOrderWithSignature(address,(address,bytes32,bytes),bytes,bytes32[])`: `26e0a196`
+ * 
+ * @param code the contract's deployed bytecode as a hex string
+ * @returns A boolean indicating if the contract likely implements the interface
+ */
+export function compatCheck(code: string): boolean {
+  const signatures = [
+    'cabinet(address,bytes32)',
+    'getTradeableOrderWithSignature(address,(address,bytes32,bytes),bytes,bytes32[])'
+  ]
+
+  const composableCow = ComposableCoW__factory.createInterface();
+  for (const signature of signatures) {
+    const sighash = composableCow.getSighash(signature);
+    if (!code.includes(sighash.slice(2))) {
+      return false;
+    }
+  }
+
+  return true;
```

Comment:
Just a fun little refactor suggestion, to be functional - in the functional programming sense:

```suggestion
  return signatures.every((signature) => {
    const sighash = composableCow.getSighash(signature)
    
    return code.includes(sighash.slice(2))
  })
```

Reply:
Slick :+1: